modules/data-table/test/spec.coffee

```diff
@@ -1224,7 +1241,7 @@ describe 'data-table', ->
         testTable {tableOptions}, done, (container, dt, options, data) ->
           container.select('.hx-sticky-table-wrapper').select('tbody').select('tr').selectAll('td').forEach (cell, index) ->
             if index is 0
-              cell.attr('style').should.equal('max-width: 10px; width: 10px; min-width: 10px; ')
+              cell.attr('style').should.equal('max-width: 10px; width: 10px; min-width: 10px;')
```

Comment:
The additional space here was one of the caveats for `phantomjs` that is no longer an issue